### ISSUE TITLE
ps in host should display PG connection information

### ISSUE THREAD

**Issue by jeffery9** [odoo/docker#383]
when `ps -axf` on host, it should display postgresql connection string, this is a very bad idea. this should cause the whole database be delete by the bad man

I think that entrypoint always read database connection string from config file. If want pass those information from environment variable , but always save it to config file.

**Comment by lathama** [comment]
@jeffery9 Yup, I think this is mentioned in several issues. Need time and resources. Today you can make a customer entrypoint.sh and allow odoo to read odoo.conf to keep things off the logs.

**Comment by lathama** [comment]
Related to https://github.com/odoo/docker/issues/334
